ndcube/ndcube.py

```diff
@@ -85,18 +85,19 @@ def extra_coords(self):
         pass
 
     @abc.abstractmethod
-    def crop_by_coords(self, lower_left_corner, dimension_widths):
+    def crop_by_coords(self, min_coord_values, interval_widths):
         """
-        Crops an NDCube given a lower left corner and widths of region of interest.
+        Crops an NDCube given minimum values and interval widths along axes.
 
         Parameters
         ----------
-        lower_left_corner: `list` of `astropy.units.Quantity`
-            The lower left corner of the region of interest described in physical units
-            consistent with the NDCube's wcs object.  The length of the iterable must
-            equal the number of data dimensions and must have the same order as the data.
+        min_coord_values: `list` of `astropy.units.Quantity`
```

Comment:
What about the reductive case where all the axes are the same unit so this could be one `Quantity`? Is that supported?

Reply:
It's not supported.  You need to be explicit which in the reductive case is a bit more work but it clearer and always behaves the same.